katas/pokerhands/bryan/whoWins.test.js

```diff
@@ -0,0 +1,530 @@
+import {whoWins} from './whoWins';
+import {readHand} from './pokerhands';
+
+test ('Check for black to win the hands without a tie on the sets', () => {
```

Comment:
Tests are also kind of big, try to think a better way to have more specific tests that can be grouped in smaller blocks